server.go

```diff
@@ -3876,18 +3989,15 @@ func (s *server) InboundPeerConnected(conn net.Conn) {
 	defer s.mu.Unlock()
 
 	// If the remote node's public key is banned, drop the connection.
-	shouldDc, dcErr := s.authGossiper.ShouldDisconnect(nodePub)
-	if dcErr != nil {
-		srvrLog.Errorf("Unable to check if we should disconnect "+
-			"peer: %v", dcErr)
-		conn.Close()
-
-		return
-	}
+	access, err := s.assignPeerPerms(nodePub)
```

Comment:
Nit: I would call it peerRelationship, because every returned access will have access (restrictive, temporary, protective) ?

Reply:
don't think peerRelationship is descriptive enough